appveyor.yml

```diff
@@ -15,8 +15,8 @@ environment:
     - nodejs_version: "5"
 
 install:
-  - IF %nodejs_version% EQU 0.10 npm -g install npm@2
-  - IF %nodejs_version% EQU 0.10 set PATH=%APPDATA%\npm;%PATH%
+  - npm -g install npm@latest
```

Comment:
are you sure you want to be using npm 3?  It's so much slower, and appveyor is already extremely slow.